scripts/google_cloud_storage.py

```diff
@@ -28,7 +28,7 @@ def get_files(self, from_date, output_dir):
 
         for blob in self.client.list_blobs(self.bucket_name):
 
-            if from_date and blob.time_created < from_date:
+            if from_date and blob.time_created < from_date or blob.name.endswith("/"):
```

Comment:
This looks odd, and I found it hard to comprehend due to the mixing of types (i. date comparison and str checking) Can this not be done in the for loop?

Reply:
So that `blob.name.endswith("/")` check is moved higher up one level where I create list of blobs, no such check in `for` loop anymore. Thanks!

Reply:
it's all been moved around as we discussed! thanks!